capture_hdr_loop.py

```diff
@@ -1,68 +1,65 @@
-"""
-Capture HDR frames in a loop (while actively changing some HDR settings).
+"""Capture HDR frames in a loop (while actively changing some HDR settings).
+
+Three sets of settings are chosen. These settings provide a dynamic range of
+about 9-10 stops, with a decent low- and highlight frame which is sufficient
```

Comment:
typo dash? 'low-' Or did you want to say low-light / lowlight?

Reply:
I shouldn't speak for Theie, but I believe the point here is that the HDR includes both a lowlight and a highlight frame? I do agree that it sounds like there's one frame that seems to capture both highlights and lowlights (which sounds counter-intuitive)...